data/external/scrapers/nat.py

```diff
@@ -171,10 +158,9 @@ def _get_base_room_infos():
             work_queue = new_queue
 
     total_hits = _join_room_hits()
-    _write_cache_json(cache_name, total_hits)
-    if undownloadable:  # down here to make shure, that tdtm has flushed the output
+    if undownloadable:  # down here to make sure, that tdtm has flushed the output
         _report_undownloadable(undownloadable)
-    return total_hits
+    return total_hits  # noqa: R504
```

Comment:
What does this mean?

Reply:
Thsis is here to tell flake8, that the warning `unnecessary variable assignment before return statement` should be ignored